docs/UserGuide.md

```diff
@@ -130,85 +138,94 @@ Format: `list`
 
 Examples:
-* `find n/John` returns `john` and `John Doe`
-* `find id/s894219J phone/ address/` returns phone and address of patient with IC number s894219J
-* `find id/S872D` returns `Alex Yeoh`, with IC number `S872D` <br>
-  ![result for 'find id/S872D'](images/findidS872DResult.png)
+ * `find n/John` returns `john` and `John Doe`
+ * `find id/s894219J phone/ address/` returns phone and address of patient with IC number s894219J
+ * `find id/S872D` returns `Alex Yeoh`, with IC number `S872D` <br>
+   ![result for 'find id/S872D'](images/findidS872DResult.png)
 
 ### Deleting a person or field: `delete`
 
 Deletes the specified person or the fields for the person from HealthSync
 
 Patient information includes:
-- Patient’s contacts
-- Patient’s medical history
-- Patient’s ward information
-- Patient’s upcoming appointment time(s)
+ - Patient’s contacts
+ - Patient’s medical history
+ - Patient’s ward information
+ - Patient’s upcoming appointment time(s)
 
 Format: `delete n/NAME or id/IC_NUMBER [field]`
 
-* Deletes the person with the specified `n/NAME or id/IC_NUMBER`.
-* The name or ic number must be a valid input
-* To delete a specified field only instead of the entire person, we indicate the field behind of the identification
-* If multiple people has the same name, HealthSync will display a list of people with that name together with their IC number.
+ * Deletes the person with the specified `n/NAME or id/IC_NUMBER`.
+ * The name or ic number must be a valid input
+ * To delete a specified field only instead of the entire person, we indicate the field behind of the identification
+ * If multiple people has the same name, HealthSync will display a list of people with that name together with their IC number.
 
 Acceptable values for each parameter:
-* String
+ * String
 
 Expected outputs when the command succeeds:
-* `Patient n/NAME or id/IC_NUMBER has been removed from the database`
-* `The [field] of Patient n/NAME or id/IC_NUMBER has been removed from the database`
+ * `Patient n/NAME or id/IC_NUMBER has been removed from the database`
+ * `The [field] of Patient n/NAME or id/IC_NUMBER has been removed from the database`
 
 Expected output when the command fails:
-* `Error code’s message (i.e. Invalid NRIC/ Invalid Field(s) / Database Error) `
+ * `Error code’s message (i.e. Invalid NRIC/ Invalid Field(s) / Database Error) `
 
 Examples:
-* `Delete id/S9987362H` deletes all the details of the person with the specified IC number from HealthSync.
-* `Delete n/John Doe` deletes all the details of John Doe from HealthSync.
-* `Delete n/John Doe /PHONE_NUMBER` deletes John Doe phone number from his profile.
+ * `Delete id/S9987362H` deletes all the details of the person with the specified IC number from HealthSync.
+ * `Delete n/John Doe` deletes all the details of John Doe from HealthSync.
+ * `Delete n/John Doe /PHONE_NUMBER` deletes John Doe phone number from his profile.
```

Comment:
I think this should be `p/` instead of `/PHONE_NUMBER` since using `p/` is specifying the field to be deleted.